pkg/network/protocols/http/etw_http_service.go

```diff
@@ -571,12 +571,36 @@ func httpCallbackOnHTTPRequestTraceTaskRecvReq(eventInfo *etw.DDEtwEventInfo) {
 	// userData := goBytes(unsafe.Pointer(eventInfo.Event.UserData), C.int(eventInfo.Event.UserDataLength))
 
 	// Check for size
-	if eventInfo.Event.UserDataLength < 36 {
-		parsingErrorCount++
-		log.Errorf("*** Error: ActivityId:%v. User data length for EVENT_PARAM_HttpService_HTTPRequestTraceTaskRecvReq_IP4 is too small %v\n\n",
-			etw.FormatGuid(eventInfo.Event.ActivityId), uintptr(eventInfo.Event.UserDataLength))
-		return
-	}
+	/*
+			 * WARNING
+			 *
+			 * the format of the UserData structure seemed to magically change for Server 2022
+			 * So the expected UserDataLength is 34 (or 44 for ipv6) for 22, and 36/46 for <= 2019
```

Comment:
Should we also add something along the lines "that Windows Performance Analyzer also fails to parse this particular event and we have file Microsoft bug report. We did not find other "magical" changes for this ETW provider".